server/target/const.py

```diff
@@ -0,0 +1,3 @@
+BOARDS_PATH = "/home/pi/openocd/tcl/target"
```

Comment:
I'd rather prefer this file to be called 'constants.py'

Reply:
renamed